me.dusansimic.DynamicWallpaper.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "me.dusansimic.DynamicWallpaper",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "me.dusansimic.DynamicWallpaper",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=xdg-data"
```

Comment:
Can you explain the xdg-data permission?

Reply:
Yes. App needs it to access `.local/share/{backgrounds, gnome-background-properties}` directories where the wallpapers and wallpaper definition files are stored respectively.

Reply:
Can this be limited to `xdg-data/backgrounds` and `xdg-data/gnome-background-properties` then?

Reply:
Done!